proto/wg/cosmo/platform/v1/platform.proto

```diff
@@ -1988,6 +2003,42 @@ message ConfigureNamespaceLintConfigResponse {
   Response response = 1;
 }
 
+message EnableGraphPruningForTheNamespaceRequest {
+  string namespace = 1;
+  bool enableGraphPruning = 2;
+}
+
+message EnableGraphPruningForTheNamespaceResponse {
+  Response response = 1;
+}
+
+message GraphPruningConfig {
+  string ruleName = 1;
+  LintSeverity severityLevel = 2;
```

Comment:
Should we use enum here?

Reply:
For LintSeverity? if yes, its already an enum